.ci/Jenkinsfile

```diff
@@ -10,17 +10,16 @@ class Globals {
     static extensionVersion = "dev-master"
 
     static phpVersion = "7.1"
-    static phpunitVersion = "^5.7"
     static pimRepoUrl = [
-        ce: "https://github.com/akeneo/pim-community-standard.git",
-        ee: "https://github.com/akeneo/pim-enterprise-standard.git"
+        ce: "https://github.com/akeneo/pim-community-dev.git",
+        ee: "https://github.com/akeneo/pim-enterprise-dev.git"
```

Comment:
:cry: We should always test with standard version. But I understand it can be a CI limitation.

Reply:
Not even a CI limitation, all test-related stuff has been moved to autoload-dev in pim-community-dev's composer.json (which is good), but it means we cannot use it anymore from the standard edition (it's not autoloaded anymore), even in a local env.